lib/models/todo_list.dart

```diff
@@ -80,12 +82,20 @@ ORDER BY created_at
     await db.execute('DELETE FROM lists WHERE id = ?', [id]);
   }
 
+  /// Find list item.
+  static Future<TodoList> find(id) async {
+    final results = await db.getAll('SELECT * FROM lists WHERE id = ?', [id]);
```

Comment:
You can actually just use `db.get()` for this case - that automatically does the `.first` part.